### DISCUSSION TITLE
How to determine anti-correlated genes

### DISCUSSION THREAD

**Discussion by sneddonucsf** [GreenleafLab/ArchR#1054] [Questions / Documentation]
Hello, 

I am interested in finding genes that have an anti-correlation to each other in respects to their GeneScoreMatrix and GeneIntegrationMatrix. Essentially, I am trying to find genes or TFs that are maybe accessible but not expressed, or expressed but not accessible. Is there a way to do this with ArchR? \

Thank you!

**Comment by rcorces** [comment]
Maybe you're looking for `correlateMatrices()`:
https://www.archrproject.com/reference/correlateMatrices.html
